fs/filebench.py

```diff
@@ -68,7 +68,7 @@ def test(self):
                                      'workloads', testfile)
         cmd = '%s -f %s' % (binary_path, testfile_path)
         out = process.system_output(cmd)
-        self.log.info("result:" + out)
+        self.log.info(b"result:" + out.decode())
```

Comment:
isn't `out.encode()` better ?

Reply:
@PraveenPenguin why is a **byte** `b"result"`appended with a **string** `out.decode()`. Isn't one change sufficient here? Only either of those needs to be converted anyways.

Reply:
@harish yes you are right , will change and push the changes